install/roles/prep/tasks/main.yml

```diff
@@ -1,5 +1,10 @@
 ---
 
+- name: Checking if server_version is defined when install_offline is true
+  fail:
+  when:
+    - install_offline is defined and server_version is not defined
```

Comment:
What happens if `install_offline` is defined but is set to false?

Reply:
Fixed.